packages/openapi-generator/src/generator.spec.ts

```diff
@@ -192,16 +193,13 @@ describe('generator', () => {
 
       const actual = readFile('existingConfig', 'utf8');
       await expect(actual).resolves.toMatch(endsWithNewLine);
-      await expect(actual).resolves.toMatchInlineSnapshot(`
-              "{
-                \\"inputDir/spec.json\\": {
-                  \\"packageName\\": \\"customName\\",
-                  \\"directoryName\\": \\"customName\\",
-                  \\"serviceName\\": \\"customName\\"
-                }
-              }
-              "
-            `);
+      expect(JSON.parse(await actual)).toEqual({
```

Comment:
I know we have talked about this, but I did not look at the bigger picture back then.
This test actually should remain as it was and the generator should be adjusted, so that the paths look the same for all OS.

Reason:
If this file is created when generating with certain options on Mac it should result in the same output as on Windows, so that users don't play ping pong between PRs.

This applies to all changed tests in this file.

Reply:
I updated the `getRelativePath(absolutePath: string)` method. It now returns the relative path with posix delimiters. This path gets saved in the options-per-service json, so the output of the file will now be same on both OS. 

Therefore, I also did not change `getOriginalOptionsPerService` method (where we read existing options) assuming that the file will have `/` separated paths, since we save them that way now. 

Is this assumption correct? Or should I still add a check to make sure if we read an existing file with `\\` , it is processed correctly?